chart/compass/charts/director/config.yaml

```diff
@@ -15,6 +15,8 @@ graphql:
     integrationSystems: ["integration_system:read"]
     viewer: []
     tenants: ["tenant:read"]
+    automaticScenarioAssignments: ["scenario_assignment:read"]
```

Comment:
Maybe we could think about a different name for this scope because there might be an assumption that the user without `scenario_assignment:write` scope can't assign scenarios, while it's still possible for him by setting the label. I was thinking about something that highlights the `automatic` part, eg. `automatic_scenario_assignment` or just `automatic_scenario`?

Reply:
fair enough will go with `automatic_scenario_assignment`